CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+Thank you for considering making contributions to Fourth State's Plasma MVP implementation! We welcome contributions from anyone! See the [open issues](https://github.com/FourthState/plasma-mvp-sidechain/issues) for things we need help with!
+
+Contribute to design discussions and conversation by joining the #plasma-dev channel on Blockchain @ Berkeley's [public slack](https://www.berkeleyblockchain.slack.com/).
+
+## Prerequisites
+* [Golang](https://golang.org/doc/install)
+
+* [dep](https://github.com/golang/dep)
+
+## How to get started:
+
+Add this repository into your $GOPATH/src/github.com/FourthState directory:
+
+`go get "github.com/FourthState/plasma-mvp-sidechain"`
+
```

Comment:
Also run dep ensure to get the dependencies

Reply:
Updated